filesystem/usr/local/share/landoop/sample-data/group_logviewer.yaml

```diff
@@ -0,0 +1,17 @@
+name: logviewer
+description: "logviewer"
+applicationPermissions:
+- ViewKafkaConsumers
+- ViewConnectors
+- ViewSQLProcessors
+- ViewSchemaRegistry
+- ViewTopology
```

Comment:
Please remove SchemaRegistry as part of the demo

Reply:
Done